integrations/langfuse/tests/test_tracing.py

```diff
@@ -77,3 +103,39 @@ def test_tracing_integration(llm_class, env_var, expected_trace):
         # check if the trace contains the expected user_id
         assert "user_42" in str(res.content)
         break
+
+
+def test_pipeline_serialization():
+    """Test that a pipeline with secrets can be properly serialized and deserialized"""
```

Comment:
If we expect the envs to be set for this unit test, let's use monkeypatch.setenv, for OPENAI_API_KEY too.

Reply:
Yes here it makes sense as it is unit test